### PR TITLE
-wa command working successfully now

### PR DESCRIPTION
**Pull request by SinghHimadri**
## Issue that this pull request solves
#347 
Closes: # (issue number)
#347 
## Proposed changes

Brief description of what is fixed or changed
Error removed command working successfully and changing about now. Also readme has been updated correctly. 
## Types of changes

_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

## Screenshots

Please attach the screenshots of the changes made in case of change in user interface
Screenshot from 2020-05-20 15-42-34

## Other information

Any other information that is important to this pull request

### CODE REVIEW

File: wplay/about_changer.py [hunk 1]
```diff
@@ -5,14 +5,16 @@
...
 
 from wplay.utils.helpers import whatsapp_selectors_dict
 from wplay.utils import browser_config
 # endregion
 
-load_dotenv()
-newsapi = NewsApiClient(api_key=os.getenv("NEWS_API_KEY"))
+'''
+Visit https://newsapi.org/ to get your own API key.
+'''
```
Comment (rpotter12) [inline wplay/about_changer.py:15]: Codacy Issue found: [String statement has no effect](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5556645)

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5556645&bid=18223117)

**Comment by SinghHimadri** [comment]
@rpotter12 Please review my PR.

**Comment by SinghHimadri** [comment]
> @SinghHimadri This software is a pypi package. So when a user install it from the command `pip install wplay`, the user will not able to paste the key in the script. The changes which you have made are only for those users who are running it from source code. :)

It was actually already present in the code I just removed the error it was not working 😅

**Comment by SinghHimadri** [comment]
Is there a way that a API key can be pasted itself by some script?

**Comment by SinghHimadri** [comment]
> @SinghHimadri This software is a pypi package. So when a user install it from the command `pip install wplay`, the user will not able to paste the key in the script. The changes which you have made are only for those users who are running it from source code. :)

The file which is already present requires API key but even after adding API Key the code is showing error due to os.getenv() function. I have just removed that error.Web scrapping is not possible for this website as it requires captcha.Only thing that can be done(according to me) is making a list of eg: 10 keys and randomly placing it for the user. The current file doesn't work I have just removed that error.

**Comment by rpotter12** [comment]
@SinghHimadri May be we can ask user to enter api key in terminal for now. What do you say??

**Comment by priyansh19** [comment]
We can do one thing .. add a .config file and when the user runs the command the file will be checked if the API keys are not present then we will displays the steps for adding them 😅

**Comment by priyansh19** [comment]
Or we can directly take input from CLI also but it will be a bit insecure way of doing so

**Comment by SinghHimadri** [comment]
@rpotter12 I have done the changes please see.

**Comment by SinghHimadri** [comment]
@rpotter12 is their some issue left in this PR? I have done the changes you asked.

**Pull request merged**